test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -5505,10 +5487,10 @@ BOOST_AUTO_TEST_CASE(pass_dynamic_arguments_to_the_base_base)
 			uint public m_i;
 		}
 		contract Base1 is Base {
```

Comment:
This test uses argument overwriting - I changed it to a test where arguments are passed on multiple times, I think this was more or less the intention.